modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-evaluator-impl/src/test/java/com/liferay/dynamic/data/mapping/form/evaluator/internal/function/PastDatesFunctionTest.java

```diff
@@ -45,6 +47,19 @@ public void testApplyTrue() {
 		Assert.assertTrue(
 			_pastDatesFunction.apply(
 				_todayLocalDate.toString(), _todayLocalDate.toString()));
```

Comment:
Curiosity: What happens if we compare pastDate with pastDate?

Reply:
The same thing as with _todayLocalDate and _todayLocalDate. It returns true.